src/app/views/schema-editing/graph-edit-graph/graph-edit-graph.component.ts

```diff
@@ -145,15 +147,16 @@ export class GraphEditGraphComponent implements OnInit, OnDestroy {
 
     private getPlacements() {
         this._crud.getGraphPlacements(this.graphId).subscribe(res => {
-            console.log(res);
             this.dataPlacements = <GraphPlacements>res;
-            if (this.dataPlacements.exception) {
-                // @ts-ignore
-                this._toast.exception({
-                    error: this.dataPlacements.exception.detailMessage,
-                    exception: this.dataPlacements.exception
-                });
-            }
+	    // This happens when we switch the namespace to a non-graph one on the sidebar,
+	    // because it takes time for the parent component to replace us with the correct one.
+            //if (this.dataPlacements.exception) {
+            //    // @ts-ignore
+            //    this._toast.exception({
+            //        error: this.dataPlacements.exception.detailMessage,
+            //        exception: this.dataPlacements.exception
+            //    });
+            //}
```

Comment:
Is this still required?

Reply:
I just kept in, so it is clear why we are not displaying an error message.  We can also delete it or just keep the first two lines.